migtests/tests/mysql/reserved-words/ReservedWords.sql

```diff
@@ -52,4 +54,39 @@ INSERT into `check` values(1, 'abc');
 INSERT into `check` values(2, 'abc');
 INSERT into `check` values(3, 'abc');
 INSERT into `check` values(4, 'abc');
-INSERT into `check` values(5, 'abc');
\ No newline at end of file
+INSERT into `check` values(5, 'abc');
+
+drop table if exists `Mixed_Case_Test`;
+
+CREATE TABLE `Mixed_Case_Test` (

```

Comment:
Assuming this is mixed case just for the table name and not the columns, can you rename it to `Mixed_Case_Table_Name_Test` or something like that just to make it more explicit?